services/version-api/helpers/getApplicationUpdateStatus.js

```diff
@@ -0,0 +1,34 @@
+import smCompare from 'semver-compare';
+
+import { VERSION_STATUS } from '../constants';
+
+const COMPARISON_RESULT = {
+  EQUAL: 0,
+  SMALLER: -1,
+  GREATER: 1,
+};
+
+/**
+ * @param {{current: string, min: string, max: string}} parameters (properties in semver format i.e "1.2.0")
+ * @returns {string} Status
```

Comment:
`Status` doesn't really say that much. Could we expand this to list the possibilities, or better scope it from `string` to `VERSION_STATUS` constant? (not sure if that is possible with jsdoc, probably not).

Reply:
You cannot have a javascript object as type, so will manually typing the return strings. This will also be changed later on when we start using typescript.